### PR TITLE
Bibs

### PR DESCRIPTION
**Pull request by pancetta**

### CODE REVIEW

File: _bibliography/pint.bib [hunk 1]
```diff
@@ -951,6 +951,15 @@ @article{MadayEtAl2007
 	year = {2007},
 }
 
+@inbook{Ulbrich2007,
+	author = {Stefan Ulbrich},
+	title = {7. Generalized SQP Methods with ``Pararea'' Time-Domain Decomposition for Time-Dependent PDE-Constrained Optimization},
```
Comment (danielru) [inline _bibliography/pint.bib]: There is an `l` missing in Parareal.

**Comment by pancetta** [comment]
mehmehmeh

File: _bibliography/pint.bib [hunk 2]
```diff
@@ -951,6 +951,15 @@ @article{MadayEtAl2007
...
 
+@inbook{Ulbrich2007,
+	author = {Stefan Ulbrich},
+	title = {7. Generalized SQP Methods with ``Parareal'' Time-Domain Decomposition for Time-Dependent PDE-Constrained Optimization},
+	booktitle = {Real-Time PDE-Constrained Optimization},
+	chapter = {},
+	pages = {145-168},
+	doi = {10.1137/1.9780898718935.ch7},
+}
+
```
Comment (danielru) [inline _bibliography/pint.bib]: For some reason, this entry does not show up when I test locally. Can you confirm it is correctly generated for you?

**Comment by pancetta** [comment]
mehmehmehmeh

See? That's the reason why you need to do this..

File: _bibliography/pint.bib [hunk 3]
```diff
@@ -951,6 +951,16 @@ @article{MadayEtAl2007
 	year = {2007},
 }
 
+@inbook{Ulbrich2007,
+	author = {Stefan Ulbrich},
+	title = {7. Generalized SQP Methods with ``Parareal'' Time-Domain Decomposition for Time-Dependent PDE-Constrained Optimization},
+	booktitle = {Real-Time PDE-Constrained Optimization},
+	chapter = {},
+	pages = {145-168},
+	doi = {10.1137/1.9780898718935.ch7},
```
Comment (danielru) [inline _bibliography/pint.bib]: Now add `url = {https://dx.doi.org/10.1137/1.9780898718935.ch7},` and we are good to go :)

**Comment by pancetta** [comment]
Phew!

**Pull request merged**